src/pages/index.tsx

```diff
@@ -1,11 +1,15 @@
-import Head from 'next/head';
+import { useState } from 'react';
+
+// eslint-disable-next-line import/extensions
+import RichTextEditor from '../components/RichTextEditor';
 
 export default function Home() {
+  const [editorContent, setEditorContent] = useState('');
+
+  const handleSubmit = (htmlContent: string) => {
+    setEditorContent(htmlContent);
+    // console.log('Content submitted:', htmlContent);
+  };
```

Comment:
if this function is setting the editor content in the state above, that is alright, but you just have to change the name to be `handleChange` or something like that